src/main/scala/org/coroutines/Coroutine.scala

```diff
@@ -198,15 +214,36 @@ object Coroutine {
       $result
     }
 
+    /** Returns whether or not the coroutine completed without an exception.
+     *
+     *  @return `true` if the coroutine completed without an exception, `false`
+     *          otherwise.
+     */
     final def hasResult: Boolean = isCompleted && $exception == null
 
+    /** Returns an `Option` instance wrapping this coroutine's non-exception
```

Comment:
On a single line, this would become a line of length 92. I'll make the line length closer to 88 though.